### PR TITLE
Validating Form using JS

### PR DESCRIPTION
**Pull request by lucabenaventew**
# Project Updates :green_circle: 
- [x] The user gets notified to insert his name if it's missing.
- [x] The user gets notified when uppercase is used in email input.
- [x] If the message is sent from a phone it will appear a text that says "Sent from my mobile phone"

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @lucabenaventew, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from both the members in your pair programming group. Make sure all the members contribute to the repo. You can refer below for submission guidelines for the project.

Screenshot 2022-06-16 at 7 04 16 AM

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @lucabenaventew, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from both partners in your group. Kindly contact student success if your partner is not responding.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @lucabenaventew, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## You have submitted a project, that doesn't have commits from both partners in your group. Kindly contact student success if your partner is not responding. Don't submit the project again and again. Kindly follow the rules of microverse and submission method.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: Scripts/form_validation.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      error[3].textContent = '';
+    }
+  }
+
+  if (data[1].value.toLowerCase() !== data[1].value) {
+    e.preventDefault();
+    data[1].classList.add('error');
+    error[1].textContent = 'Email cant contain upercase';
+    error[3].textContent = 'Email cant contain upercase';
+    e.preventDefault();
```
Comment (danmainah) [inline Scripts/form_validation.js:30]: - [ ] Kindly ensure your error message only appears near the submit button only, it appears twice:
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @lucabenaventew  and @Alzubair98 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: Scripts/form_validation.js (hunk 1 above)
Comment (lucabenaventew) [reply Scripts/form_validation.js]: For the code reviewer that just rejected my code without any comment, here is the proof that I fixed the previous reviewer requirements.

image

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @lucabenaventew 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by OLIPLICHE — COMMENTED** [review]
Hi @raminka13 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

Linter result is green ✔️
 Commit messages are meaningful✔️
 The pull request have a descriptive title telling you what changes are inside ✔️
 The pull request have a short summary describing introduced changes in more detail ✔️
 node_modules folder is in the .gitignore ✔️
Required Changes ♻️
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OLIPLICHE** [comment]
- [ ] Hi, Sorry that I had a Technical issue at my end. my comment above was the reason why I did not approve the Project. Nice Job that you have implemented the changes and got approved. Sorry for delaying your time. 
Best,

**Pull request merged**